.luacheckrc

```diff
@@ -49,5 +49,7 @@ read_globals = {
     'UnitPowerMax',
     'UnitPowerType',
     'UnitRace',
+    'ArenaEnemyFramesContainer',
```

Comment:
This looks good. Can you put these at the top of the file? They seem to be in alphabetical order :)